docs/dev.md

```diff
@@ -0,0 +1,15 @@
+#### dev
+
+----
+Start a Liberty server in dev mode.
+
+
```

Comment:
Also provide an Additional Parameters section similar to the one [here](https://github.com/WASdev/ci.maven/blob/master/docs/create-server.md#additional-parameters)
`The following are the parameters supported by this goal in addition to the [common server parameters](common-server-parameters.md#common-server-parameters) and the [common parameters](common-parameters.md#common-parameters).`